.sqlx/query-1fe4eb033e38d873d0a875093c4bfaea0f8d02c8a70744bc76cb067fee6e3a5c.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Ohh!! I like that you generated this file!

Just wanted to explain what it is and why we will want!

`sqlx` is a bit 'special' in that when you compile the `query!` macro it actually connects to the Postgres database and checks what tables and columns exist and all that. Which is really cool cause it can provide us with more type safety!

But the reason we want/need this file is that its kinda annoying to NEED a postgres database to be able to compile this code. On our laptops its not much of an issue, but when we want to deploy this code to a web server, or run tests in CI sometimes its more convenient to not need Postgres setup

So if we set the `SQLX_OFFLINE=1` then sqlx will use this instead of the DB to check the schema.

You already did it once but to type it out this is generated with `cargo sqlx prepare --workspace` (the `--workspace` bit is nice since we have the `db` and `cli` crates)

Reply:
Awesome! That's very cool!